base/qaqc/DESCRIPTION

```diff
@@ -43,4 +44,4 @@ LazyData: FALSE
 Encoding: UTF-8
 VignetteBuilder: knitr
 Config/testthat/edition: 3
-RoxygenNote: 7.1.2
+RoxygenNote: 7.2.3
```

Comment:
Sorry about this undocumented requirement, but: Can you please rebuild this using Roxygen 7.1.3? You can install it by running `remotes::install_version("roxygen2", "7.1.3")`, then manually edit this line back to "7.1.3" (Roxygen doesn't run if it thinks a newer version touched the package last) and run Roxygen again.

Why are we picky about making everyone use an outdated Roxygen version? That's a fair question. Our reasoning for making everyone use the same version is that If different contributors are using different versions of Roxygen, then redocumenting after one change can wind up introducing spurious changes in another place you didn't touch, and then another contributor with a different version of Roxygen tries to redocument their work and winds up reverting your change, and so on.

We are due for a bump in our version requirement so that we insist everyone use Roxygen 7.2.3 instead of Roxygen 7.1.3, but we haven't gotten around to doing it yet. If anyone reading this wants to take on this version bump, see #2685 for an outline of the approach.

Reply:
Thanks for the very clear explanation. It totally makes sense. 
Well actually there is no `version 7.1.3` for `roxygen2`. Checked releases and also running the suggested command gives 

> version '7.1.3' is invalid for package 'roxygen2'

[roxygen2-releases](https://github.com/r-lib/roxygen2/releases)

Reply:
🤦 I meant 7.1.2, like in the old version of this line, but apparently I carried the typo through all four times I wrote it out... maaaaaybe I need to start wearing glasses at the computer.